src/modules/navigator/rtl.h

```diff
@@ -77,22 +79,22 @@ class RTL : public MissionBlock, public ModuleParams
 
 	void find_RTL_destination();
 
-	void set_return_alt_min(bool min);
+	void set_return_alt_min(bool min) { _rtl_alt_min = min; }
 
-	int rtl_type() const;
+	int rtl_type() const { return _param_rtl_type.get(); }
```

Comment:
Please split minor changes from actual changes in separate commits. It makes reviewing easier, thanks.